### DISCUSSION TITLE
Labeling known addresses ( Pool address, Exchange address, etc )

### DISCUSSION THREAD

**Discussion by ghost** [mempool/mempool#1840] [Ideas]
It would be good if we could classify known bitcoin addresses and show them at frontend.

**Comment by BLdotN** [comment]
Although i am not a big fan of classifying adresses to identities... The reality is that it's possible and that it's already being done. Mainly by professional parties that sell that information outside the reach of plebs. So...

This would be really usefull and nice to have! To extend on the idea, it would be nice to:

-  Be able to import lists of classified addresses 
-  Add a (optional) prediction score '%' to a tag. 
-  REST endpoints to update the values.

This would make it a lot easier for the selfhosted/opensource plebs to visualize this kind of information (locally). And to write scripts to automatically update the (predicted) tags.
